### ISSUE TITLE
improvements

### ISSUE THREAD

**Issue by droshev** [fundamental-conf/fundamental-conf#6]
- there are 2 pages loaded separately if u are on mobile or on desktop and they are reloading the one or the other on 800px viewport width
- landmarks are missing like <main>, <header>, <footer> etc. (big a11y issue)
- headline structure is completely missing
- navigation on desktop is not a <nav> and ul li …same for slidein navigation on mobile (not accessible at all)
- images with PNG instead of SVGs with inline code
- very bad or missing alt attributes for images
- hidden text elements with opacity (main navigation agenda) which would end up doubled in google and screenreaders
- the branding animation letters are plain images with no alt …and are not hidden for screenreaders …also no SVGs
- semantic markup is missing for the info boxes like date and time. also the text is written in capital letters in the markup and are not styled via CSS
- Cards section is not semantic markup and not accessible. images with no alt
- speaker section is not semantic markup and is not accessible via keyboard etc.
- meaningless empty DIVs in the markup to get some whitespace between areas
- footer has no <nav> and no headline
- target=“_blank” is a bad practise for links
- CSS doesn’t use naming conventions like BEM which fundamentals is promoting
- the speakers should be in `<figure><figcaption>`
